styles/cart.css

```diff
@@ -0,0 +1,101 @@
+.search_navbar {
+  padding: 10px;
+  width: var(--width-100);
+}
+.cake-logo {
+  color: var(--color-white);
+  font-size: var(--size-7);
+}
+.fa-search {
+  position: absolute;
+  top: 43%;
+  left: 26%;
+  cursor: pointer;
+}
+.list {
+  justify-content: space-around;
+}
+.list_end {
+  /* padding-left: 10%; */
+}
+.footer {
+  display: flex;
+  bottom: 0;
```

Comment:
what is the use of bottom 0 here? using postitions?

Reply:
yeah position is absolute here